model-optimizer/extensions/front/mxnet/ssd_anchors_replacer.py

```diff
@@ -0,0 +1,147 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from typing import Dict
+
+from extensions.front.mxnet.ssd_detection_output_replacer import SsdPatternDetectionOutputReplacer
+from extensions.ops.elementwise import Div, Add, Sub
+from extensions.ops.split import Split
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.front.tf.graph_utils import create_op_node_with_second_input
+from mo.graph.graph import Graph, Node
+from mo.ops.concat import Concat
+from mo.ops.reshape import Reshape
+
+
+class SsdAnchorReshape(FrontReplacementPattern):
+    enabled = True
+    graph_condition = [lambda graph: graph.graph['fw'] == 'mxnet' and graph.graph['cmd_params'].enable_ssd_gluoncv]
+
+    def run_after(self):
+        return [SsdPatternDetectionOutputReplacer]
+
+    def pattern(self):
+        return dict(
+            nodes=[
+                ('slice_like', dict(op='slice_like')),
+                ('reshape0', dict(op='Reshape')),
+                ('reshape1', dict(op='Reshape')),
+                ('reshape2', dict(op='Reshape')),
+                ('reshape3', dict(op='Reshape')),
+                ('concat', dict(op='Concat')),
+            ],
+            edges=[
+                ('slice_like', 'reshape0'),
+                ('reshape0', 'reshape1'),
+                ('reshape1', 'reshape2'),
+                ('reshape2', 'reshape3'),
+                ('reshape3', 'concat'),
+            ])
+
+    def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
+        slice_like = match['slice_like']
+        slice_like.out_port(0).disconnect()
+        match['reshape2'].out_port(0).get_connection().set_source(slice_like.out_port(0))
+
+
+class SsdAnchorsReplacer(FrontReplacementPattern):
```

Comment:
General question. Here we use `create_op_node_with_second_input` multiple times. Second input is `int64` value here. Are we 100% sure that the first input is `int64` too? What are the options here? Should we explicitly insert a Convert?